src/javascript/tail.js

```diff
@@ -0,0 +1 @@
+})(typeof global !== "undefined" ? global : typeof window !== "undefined" ? window : this)
```

Comment:
TODO: add `globalThis` for modern browsers and `self` for web workers in older browsers

Reply:
++